app/Domain/Locations/Models/Location.php

```diff
@@ -29,10 +33,23 @@ class Location extends Model
     ];
 
     /**
-     * @phpstan-return \Illuminate\Database\Eloquent\Relations\HasMany<\App\Models\LocationImage>
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo<\App\Domain\Trips\Models\Trip, \App\Domain\Locations\Models\Location>
```

Comment:
Since belongsTo and the Trip Model is already imported can we remove the full path from these to keep it shorter and cleaner? :)

Reply:
done